src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/SearchByLastName.kt

```diff
@@ -2,11 +2,10 @@ package uk.gov.justice.digital.hmpps.welcometoprison.model.prison.prisonersearch
 
 import java.time.LocalDate
 
-data class SearchByNameAndDateOfBirth(
-  val firstName: String,
+data class SearchByLastName(
   val lastName: String,
-  val dateOfBirth: LocalDate,
+  val firstName: String? = null,
+  val dateOfBirth: LocalDate? = null,
   val pncNumber: String? = null,
-  val croNumber: String? = null,
   val nomsNumber: String? = null
 )
```

Comment:
this makes lastName mandatory?

Reply:
probably worth removing noms and pnc number too

Reply:
lets talk about this